frontend/src/pages/CollaborationReports/components/CollabReportAlertsTable.js

```diff
@@ -22,6 +22,10 @@ const ReportLink = ({ report, userId }) => {
     return <Link to={`/collaboration-reports/view/${report.id}`}>{report.displayId}</Link>;
   }
 
+  if (isSubmitted && isApprover) {
```

Comment:
Can you confirm for me that this is supposed to be an `&&` and not an `||`?

Reply:
There's a different template for submitted+creator, its the "view" instead of "review" link (hardly confusing at all)

Reply:
I'm not sure if there are other cases I missed though